src/Functions/seriesDecomposeSTL.cpp

```diff
@@ -0,0 +1,183 @@
+#include "config.h"
+
+#if USE_SEASONAL
+#    ifdef __clang__
```

Comment:
(cosmetic: let's remove the indentation on l. 4-25, preprocessor directives (l. 3) are no language constructs like `if`, `for` etc., and therefore don't affect indentation)